ouroboros-consensus-diffusion/test/consensus-test/Test/Consensus/PeerSimulator/BlockFetch.hs

```diff
@@ -104,12 +99,12 @@ runBlockFetchClient
   => peer
   -> FetchClientRegistry peer (Header TestBlock) TestBlock m
   -> ControlMessageSTM m
-  -> m (AnchoredFragment TestBlock)
+  -> BlockFetchServer TestBlock (Point TestBlock) m ()
   -> m ()
-runBlockFetchClient peerId fetchClientRegistry controlMsgSTM getCurrentServerChain =
+runBlockFetchClient peerId fetchClientRegistry controlMsgSTM server =
```

Comment:
Why is it called `runBlockFetchClient` when it runs both the client and the server? Isn't the equivalent for chain sync called `runChainSyncConnectionThread` or something?

Reply:
there's also `startBlockFetchConnectionThread` in `Run.hs`. I've created a corresponding file for `ChainSync`.